wiki/Article_Styling_Criteria/en.md

```diff
@@ -108,55 +114,59 @@ Folder names must not contain URL reserved characters. Generally, all folder nam
 
 #### Percent encoding characters
 
-Folder names should not use characters that require percent encoding. Those characters may include, but are not limited to:
+Folder names must not use characters that require percent encoding. Those characters may include, but are not limited to:
 
 - spaces
-  - all spaces must be underscores (`_`).
+  - all spaces must be changed to underscores (`_`)
 - apostrophes (`'`)
 
 ### Article file names
 
-The name of an article file can be found in the `File Name` column of the [locales section](#locales). The location of a translation is to be placed in the appropriate English-named folder.
+The file name of an article can be found in the `File Name` column of the [locales section](#locales). The location of a translated article must be placed in the same folder as the English version.
 
 ### Index articles
 
-All folders should contain an article of some kind. If it seems impractical to create an article for a folder that holds other articles, create an index article instead. Index articles list off articles that are inside its own folder and could contain a description for each link.
+All folders should contain an article of some kind. If it seems impractical to create an article for a folder that holds other articles, create an index article instead. Index articles list off articles that are inside its own folder and may contain a description for each link.
 
 ### Disambiguation articles
 
-[Disambiguation](/wiki/Disambiguation) articles must be placed in the `/wiki/Disambiguation` folder.
+[Disambiguation](/wiki/Disambiguation) articles must be placed in the `/wiki/Disambiguation` folder. The main page must be updated to include the disambiguation article. Refer to [Disambiguation/Mod](/wiki/Disambiguation/Mod) as an example.
+
+Redirects must be updated to have the ambiguous keyword(s) redirect to the disambiguation article.
 
 ## Markdown
 
 All osu! wiki articles must adhere to the following rules.
 
 ### GitHub Flavored Markdown
 
-Parts of the GFM (GitHub Flavored Markdown) markup are supported in the osu! wiki. GFM has a small amount of markup that anyone can learn in about 3 minutes. However, do not expect a lot out of GFM as it is intended to be used as a bare minimum markup language.
+Some parts of the GitHub Flavored Markdown markup are supported in the osu! wiki.
```

Comment:
Let's make it a bit more formal-sounding.
```suggestion
A subset of the GitHub Flavored Markdown syntax are supported in the osu! wiki.
```

Reply:
You might as also link this [article](https://help.github.com/en/articles/about-writing-and-formatting-on-github) about GFM on "GitHub Flavored Markdown" as well to give contributors an idea what GFM is.

Reply:
I forgot why I wrote that here. Does the website support any markup from GFM, other than the regular Markdown? A subset (or "some parts") of GFM seems vague or doesn't sound right if it is just Markdown.

I'll just remove the section instead, if there is no other discussion.

Reply:
Also, I think that article should be linked in the `osu! wiki contribution guide` instead.

Reply:
https://github.github.com/gfm/

Tables are not supported in regular markdown, maybe that's why.

Reply:
Markdown tables are part of the Markdown syntax spec @Joehuu

Reply:
> Also, I think that article should be linked in the `osu! wiki contribution guide` instead.

good wiki practice is that this should also be linked in the criteria page as well. It's better you link GFM's definition since one reference on one article is not enough.

Reply:
GFM as an extension, yes. Commonmark or regular markdown, no.

Reply:
they aren't part of commonmark, which is the spec that osu! uses. osu! is also using other extensions and options that aren't standard though, so that's why these things are possible

anyway i think this note about GFM is a little pointless since any info it potentially gives is still meaningless (osu! has nothing to do with GFM, even if they happen to be similar). there's plenty of info about markdown in this article already

Reply:
Yeah I think we should remove this altogether if that's the case.